go/test/endtoend/binlog/binlog_test.go

```diff
@@ -110,11 +110,11 @@ func TestMain(m *testing.M) {
 			return 1, err
 		}
 
-		srcMaster = localCluster.GetVttabletInstance("master", 0, "")
+		srcMaster = localCluster.GetVttabletInstance("replica", 0, "")
```

Comment:
It was causing a test failure due to conflicting code. I have reverted this now.

Reply:
I did that because I had removed the below code which is ideally not required.
https://github.com/vitessio/vitess/blob/master/go/test/endtoend/cluster/vtctlclient_process.go#L102-L104

But after removing the code few of intial_sharding test cases started failing. So Arindam had to revert this change. The above is one place which we forgot to revert. 

We will take that refactoring in another PR.